src/components/WebsiteTabs/website-tabs.scss

```diff
@@ -35,16 +35,22 @@
   color: $text-01;
   line-height: 1.25;
   padding: $spacing-sm $spacing-05;
+  transition: background $transition--base;
```

Comment:
Should we be pulling in and consuming the motion package? We've switched away from `$transition--base` and have been using the new `$duration` tokens. Background changes should be `fast-01` (I think). @shixiedesign can you confirm?

Reply:
Yes! We should be using `duration--fast-01` for hover animations. Thanks!